### DISCUSSION TITLE
Change text color of markdown snippet of 3 cols with dark theme

### DISCUSSION THREAD

**Discussion by EdwineMathieu** [OliBlade/3Cols#35] [Ideas]
In 3Cols, is it possible to change the color of texts in markdown snippets when using a dark theme ?

**Comment by OliBlade** [comment]
There's a couple of ways. You can use the backtick " ` " to write a code section which will give you pink text. Although this doesn't seem to color correctly when using multiline " ``` ".

Alternatively you can use html to get whatever color you like using the following: `<font color='yellow'>Yellow text</font>`

**Reply by EdwineMathieu** [reply]
Thank you so musch @OliBlade
